test/unit_tests/js/modules/util/breakpoint-state-spec.js

```diff
@@ -41,29 +47,45 @@ describe( 'breakpoint-state', () => {
 
       // eslint-disable-next-line guard-for-in
       let rangeKey;
-      for ( rangeKey in breakpointsConfig ) {
-        if ( {}.hasOwnProperty.call( breakpointsConfig, rangeKey ) ) {
-          width = breakpointsConfig[rangeKey].max ||
-                  breakpointsConfig[rangeKey].min;
-          breakpointStateKey =
-            'is' + rangeKey.charAt( 0 ).toUpperCase() + rangeKey.slice( 1 );
-
-          expect( breakpointState.get( width )[breakpointStateKey] )
+      for ( rangeKey in varsBreakpoints ) {
+        if ( {}.hasOwnProperty.call( varsBreakpoints, rangeKey ) ) {
+          width = varsBreakpoints[rangeKey].max ||
+                  varsBreakpoints[rangeKey].min;
+
+          expect( getBreakpoint( width )[rangeKey] )
             .toBe( true );
         }
       }
     } );
 
   } );
 
-  describe( '.isInDesktop()', () => {
+  describe( '.isIn()', () => {
     it( 'should determine whether inside desktop breakpoint threshold', () => {
-      expect( breakpointState.isInDesktop() ).toBe( true );
+      expect( isIn( DESKTOP ) ).toBe( true );
+
+      /* TODO:
+         Mock getBreakpoint() to return a small breakpoint size,
```

Comment:
Instead of mocking you could try [this technique](https://github.com/cfpb/cfgov-refresh/blob/fe13404b973d4d8eca996b8fd50ab66f1d1df677/test/unit_tests/js/modules/BreakpointHandler-spec.js#L9-L16).

Reply:
Works great! Updated in https://github.com/cfpb/cfgov-refresh/pull/5556/commits/ac6359c9536dd5b90d54f0150378127e5667e195